crates/solvers-dto/src/auction.rs

```diff
@@ -279,3 +281,14 @@ pub struct ForeignLimitOrder {
     #[serde_as(as = "HexOrDecimalU256")]
     pub taker_token_fee_amount: U256,
 }
+
+#[serde_as]
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct FlashloanHint {
+    pub lender: Option<H160>,
+    pub borrower: Option<H160>,
```

Comment:
I believe when passing this data to the solver it should not be optional anymore. When the driver parses the appdata it should replace `borrower` with the order owner if it's not set. And I would imagine the `lender` to be set to a sane default the driver is compatible with if it's not set. Given that we only support the Maker contracts right now we should go with that. Also might make sense to make this configurable later on.

Reply:
Switched to non-optional values.